setup.py

```diff
@@ -30,7 +32,10 @@
         'tabulate'
     ],
     url='https://github.com/dataxu/github-pr',
-    version='1.1.1',
+    version='2.0.0',
     scripts=['github_pr.py'],
     keywords=['github'],
+    dependency_links=[
+        "git+https://github.com/PyGithub/PyGithub#egg=PyGithub-1.27.0"
```

Comment:
we are going to install this from github because, while they released 1.27.0 today, it may take a while to get a PyPi update to 1.27.0